core/domain/classifier_services.py

```diff
@@ -127,6 +127,70 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+
+
+def add_to_training_queue(exploration):
+    """Checks all states of the exploration for re-training conditions and
+    creates ClassifierTrainingJobModel instances in the datastore
+    accordingly.
+
+    Args:
+        exploration: Exploration. Domain object for an exploration.
+    """
+    states = exploration.states
+    for state_name in states:
+        state = states[state_name]
+        if not state.can_undergo_classification():
+            continue
```

Comment:
IMHO, that is how I like to program (and what I was always taught in college - always avoid `break` and `continue` in loops which are not very short and succinct, it leads to less readable programs). @seanlip what do you think?